nsq/reader.py

```diff
@@ -358,7 +344,7 @@ def _maybe_update_rdy(self, conn):
         if self.backoff_timer.get_interval() or self.max_in_flight == 0:
             return
 
-        if conn.rdy <= 1 or conn.rdy < int(conn.last_rdy * 0.25):
+        if conn.rdy == 1:
```

Comment:
Right.  Updating `RDY` here used to serve two purposes:  Periodically reupping `RDY` when it dipped too low, and going from an initial throttled `RDY 1` on startup to full-throttle connection-max-in-flight.  Now only the second case is relevant.